### DISCUSSION TITLE
paopaogate定制ISO后启动报错

### DISCUSSION THREAD

**Discussion by Seed680** [kkkgo/PaoPaoGateWay#123] [问和答]
大佬 我按说明定制了ISO后，启动报错如图
我配置了paopaodns，也能打开http://192.168.5.4:7889/ppgw.ini
我是哪里做的不对吗
ini配置如下

> #paopao-gateway
> 
> # mode=socks5|ovpn|yaml|suburl|free
> # default: free
> mode=suburl
> 
> # Set fakeip's CIDR here
> # default: fake_cidr=7.0.0.0/8
> fake_cidr=7.0.0.0/8
> 
> # Set your trusted DNS here
> # default: dns_ip=1.0.0.1
> dns_ip=192.168.5.4
> # default: dns_port=53
> # If used with PaoPaoDNS, you can set the 5304 port
> dns_port=5304
> 
> # Clash's web dashboard
> clash_web_port="80"
> clash_web_password=""
> 
> # default：openport=no
> # socks+http mixed 1080
> openport=yes
> 
> # default: udp_enable=no
> udp_enable=no
> 
> # default:30
> sleeptime=30
> 
> # socks5 mode settting
> # default: socks5_ip=gatewayIP
> #socks5_ip="10.10.10.5"
> # default: socks5_port="7890"
> #socks5_port="7890"
> 
> # ovpn mode settting
> # The ovpn file in the same directory as the ppgw.ini.
> # default: ovpnfile=custom.ovpn
> #ovpnfile="custom.ovpn"
> #ovpn_username=""
> #ovpn_password=""
> 
> # yaml mode settting
> # The yaml file in the same directory as the ppgw.ini.
> # default: yamlfile=custom.yaml
> #yamlfile="custom.yaml"
> 
> # suburl mode settting
> suburl="我的机场订阅"
> subtime=1d
> 
> # fast_node=check/yes/no
> fast_node=yes
> test_node_url="https://www.youtube.com/generate_204"
> ext_node="Traffic|Expire| GB|Days|Date|倍"
> cpudelay="3000"
> fall_direct="no"
> # dns burn setting
> # depend on fast_node=yes & mode=suburl/yaml
> dns_burn=yes
> # If used with PaoPaoDNS, you can set the PaoPaoDNS:53
> ex_dns="192.168.5.4:53"
> 
> # Network traffic records
> net_rec=yes
> max_rec=50000

![image](https://github.com/user-attachments/assets/023e65be-bac2-4e35-85be-9f69d3afd40f)

**Comment by kkkgo** [comment]
你可以浏览器打开http://192.168.5.4:7889/ppgw.ini   截图一下，应该是你第一行去掉了注释报文件损坏。

**Comment by Seed680** [comment]
> 你可以浏览器打开http://192.168.5.4:7889/ppgw.ini 截图一下，应该是你第一行去掉了注释报文件损坏。

![image](https://github.com/user-attachments/assets/7999e30e-2084-4b10-a696-c0c95d191044)
大佬 这个是配置文件

**Reply by kkkgo** [reply]
#paoo，你多打了一个o

**Comment by Seed680** [comment]
> 你可以浏览器打开http://192.168.5.4:7889/ppgw.ini 截图一下，应该是你第一行去掉了注释报文件损坏。

大佬 我按你说的 删除了配置文件 重新把内容复制了一下 的确好了 感谢

**Comment by Seed680** [comment]
大佬 面板服务可以进行iso定制替换吗

**Reply by kkkgo** [reply]
你可以自己搭面板填自己的api地址。
